### DISCUSSION TITLE
How to check calico-enterprise updates

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#22986] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.140.0

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

We do wonder how to configure renovate to access quay.io repositories, where tigera docker images are hosted. 
No we do wonder how to change the _renovate.json_ to let renovate check for available updates of tigera ?:
```json
		{
			"hostType": "docker",
			"dnsCache": true,
			"enableHttp2": true,
			"keepalive": true,
			"matchHost": "quay.io"
		},
````

### What you tried so far.

We configured within our Kubernetes cluster appropriate secrets for calico-enterprise itself in this manner:
```json
{
	"auths": {
		"quay.io": {
			"auth": "<snip>",
			"email": ""
		}
	}
}
```
which works fine.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Docker images on quay.io should work already. Are you referring to a provide Docker image? If so then you'd need a hostRules entry for the host

**Reply by toralf** [reply]
And what is the syntax for the credentials?

**Reply by rarkins** [reply]
https://docs.renovatebot.com/docker/#self-hosted-docker-registry
